enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/IndexPersistenceAcceptanceTest.scala

```diff
@@ -165,7 +165,8 @@ class IndexPersistenceAcceptanceTest extends IndexingTestSupport {
 
   test("Should not get new index configuration on database settings changes of WGS84 minimum x extent") {
     // remove the entire western hemisphere
-    testIndexRestartWithSettingsChanges(Map(SpatialIndexSettings.spatial_crs_WGS_84_x_min -> "0"))
+    val wgs84_x_min = Settings.setting("unsupported.dbms.db.spatial.crs.wgs-84.x.min", Settings.DOUBLE, "0")
```

Comment:
There is no alternative, since there are no longer static constants for the dynamic settings. If you look at SpatialIndexSettings.java you will see we deleted many, many static constants in favour of much shorter code that generated them dynamically. This also makes it more future proof when we expand from more than 4 CRS.